code/modules/projectiles/projectile/special.dm

```diff
@@ -133,30 +133,6 @@
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/green_laser
 	flag = "energy"
 
-/obj/item/projectile/energy/floramut/on_hit(var/atom/target, var/blocked = 0)
```

Comment:
Replacements for these?

Reply:
This was moved to Diona's `bullet_act` species code, specifically, since the way it's currently handled is ❄️.

I opted not to include it for golems since they're semi-magical anyway and it'd mean having the same exact code copy-pasted.